src/main/java/org/folio/config/RMAPIConfiguration.java

```diff
@@ -7,6 +7,7 @@
   private String customerId;
   private String apiKey;
   private String url;
+  private Boolean isValid;
```

Comment:
can we please rename this variable to be "getConfigurationValid" instead of "isValid" for more clarity?